temporal-cluster/server/entrypoint_override.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -eu -o pipefail
+
+export BIND_ON_IP="${BIND_ON_IP:-$(hostname -i)}"
+
+if [[ "${BIND_ON_IP}" =~ ":" ]]; then
+    # ipv6
+    export TEMPORAL_CLI_ADDRESS="[${BIND_ON_IP}]:7233"
+else
+    # ipv4
+    export TEMPORAL_CLI_ADDRESS="${BIND_ON_IP}:7233"
+fi
+
+# @render override for non-frontend services.
+export PUBLIC_FRONTEND_ADDRESS="${PUBLIC_FRONTEND_ADDRESS:-${RENDER_FRONTEND_HOST}:7233}"
```

Comment:
If this is the only change to the file, maybe we should use temporal's entrypoint.sh file as is and just change the `ENTRYPOINT` command in the Dockerfile, like we do with the `temporal-ui` service.